lib/cobrato/entities/payer.rb

```diff
@@ -1,6 +1,8 @@
 module Cobrato
   module Entities
     class Payer < Base
+      puts "Warning: Payer entity is deprecated. Please use Person entity instead."
```

Comment:
I suggest to warn only when someone use the methods. See [regress_cnab.rb:9](https://github.com/myfreecomm/cobrato-client-ruby/blob/master/lib/cobrato/resources/regress_cnab.rb#L9)

Reply:
Corrigido em https://github.com/myfreecomm/cobrato-client-ruby/pull/64/commits/13e7c6c3b7c53a560df975589e984891df5c46c1